### DISCUSSION TITLE
General Question to flags and options

### DISCUSSION THREAD

**Discussion by Debo-rah** [mtex-toolbox/mtex#581] [Ask Anything]
Dear all,

is there a way to get a list of all possible flags and options of a function? I often know there must be a possibility to change something (e.g. Marker colour, or changing the grid resolution), but i dont know how to find the specific word or where and whether capital letters are used. So i just sit there and try all options that come to my mind or I edit the function and scan through it to find possible flags but depending on the function this can take quite long and if this function calls other functions its getting complicated...  
There must be an easier way? 
I know that some flags and options are in the "help" comments, but unfortunately not all of them.

Thanks!
Deborah

**Comment by kilir** [comment]
Hi,
that is true that, the help does not cover generic options, but in most cases, all those options which are important for the function should be in there. The plotting functions support (most) standard Matlab syntax (markers, lines, etc). And the best thing is, that more and more functions support TAB completion.
In case you have some important cases which are missing, you can edit the help section of the function and create a pull request.

Cheers,
Rüdiger

**Comment by ralfHielscher** [comment]
Hi Deborah,

I added a thread #582 where we collect topics that need more documentation. If you find a function where options are not sufficiently listed in the documentation could you please add this to the list?

Thank you,

Ralf.
